docs/overview/migration/MigrationV5.stories.mdx

```diff
@@ -662,3 +662,7 @@ The `theme` property was removed since the component accesses the theme object i
 ### Code editor <a id="code-editor" />
 
 The `theme` property was removed since the component accesses the theme object internally.
+
+### DatePicker <a id="date-picker" />
```

Comment:
Under the "UI Kit packages" section, we should also remove the date picker from the list of components missing for the core package

Reply:
No, look for "The following components from `@hitachivantara/uikit-react-core` are missing:" and the date picker will be there

Reply:
thanks! i missed it